test/unit-test/src/cfpb-atomic-component/src/components/AtomicComponent-spec.js

```diff
@@ -1,5 +1,4 @@
-const srcPath = require( '../src-path' );
-let AtomicComponent = require( srcPath + '/components/AtomicComponent' );
+import AtomicComponent from '../../../../../../packages/cfpb-atomic-component/src/components/AtomicComponent';
```

Comment:
What I visualize when I see relative paths like this:

![elevator](https://user-images.githubusercontent.com/1060248/98601876-a651d700-22ad-11eb-922c-0444472e0d10.gif)

It's a spec file though so no biggie.